### ISSUE TITLE
Running echinda in  docker node:14-buster hangs indefinitely.

### ISSUE THREAD

**Issue by krasi-georgiev** [crytic/echidna#633]
You can use this repo to reproduce.
https://github.com/tellor-io/chorus

Here is an example github action that was canceled after running for 6h
https://github.com/tellor-io/chorus/runs/2246207447?check_suite_focus=true

```
    docker run -it -v $(pwd):/home/krasi --entrypoint=/bin/bash node:14-buster
    apt-get update                                                                                                                                                                                 
    apt-get install -y --no-install-recommends python3-pip python3-dev build-essential
    npm install hardhat
    pip3 install crytic-compile slither-analyzer
    apt install pytho3-setuptools
    pip3 install setuptools
    pip3 install crytic-compile slither-analyzer
    wget https://github.com/crytic/echidna/releases/download/v1.7.0/echidna-test-1.7.0-Ubuntu-18.04.zip
    unzip echidna-test-1.7.0-Ubuntu-18.04.zip
    tar -xf echidna-test.tar.gz
    mv ./echidna-test /usr/bin/
    cd /home/krasi/
    echidna-test . --contract ChorusTest --config  echidna.yml
 
   echidna-test . --contract ChorusTest --format=text
   Analyzing contract: /home/krasi/contracts/ChorusTest.sol:ChorusTest <-- hangs here.
```

**Comment by elopez** [comment]
Hi! I have tested this locally and it appears to work ok; it is just slow. Looking at the fuzzing progress, I estimated it would take a bit over 2 hours to complete on my Ryzen 3900X machine, so it does not seem far-fetched that a GH Pipeline would be over 3x that time.

To confirm my observations I forked the repo and added a `testLimit` property to the `echidna.yml` file with a value of 100, on top of the ` krasi-georgiev/chorus:echidna` branch. [Here's the commit](https://github.com/elopez/chorus/pull/1/commits/0adf3f4ec5d39787ba11e271d94956fc4ca50cbd), for your reference. I then made a PR and verified that the actions complete in a reasonable time: https://github.com/elopez/chorus/runs/2254399265

![imagen](https://user-images.githubusercontent.com/2642849/113425390-2507f780-93a8-11eb-9bc4-c340c392773f.png)
